tools/java/org/unicode/cldr/api/FilteredData.java

```diff
@@ -0,0 +1,73 @@
+// © 2019 and later: Unicode, Inc. and others.
```

Comment:
Please only one copyright block, not three...

Reply:
Hehe. IntelliJ adds one on new files, but then elides it when viewing the file, so if you cut & paste all of one file into another "empty" file it's duplicated - I hadn't appreciated this failure mode before.

Reply:
not fixed yet

Reply:
Dammit - caught it for PathMatcher but not here. Done now.